test/foundry/RollupCreator.t.sol

```diff
@@ -75,7 +74,7 @@ contract RollupCreatorTest is Test {
         rollupUser = _rollupUser;
 
         //// deploy creator and set logic
-        rollupCreator.setTemplates(
+        rollupCreator = new RollupCreator(
```

Comment:
for brevity should we also programmatically verify that the contract can be frozen?